hack/csv-generate.sh

```diff
@@ -9,6 +9,7 @@ OLD="4.4.0"
 PREV="4.5.0"
 LATEST="4.6.0"
 LATEST_CHANNEL="4.6"
+CSV_SKIP_RANGE=">=4.5.0 <4.6.0"
```

Comment:
I think we should at least generate this out of `PREV` and `LATEST`. Not a fan of having it in the helper script, but I don't have bright ideas now.

Reply:
Will do, thanks